README.md

```diff
@@ -4,9 +4,64 @@ Reusable Svelte components for some internal applications
 
 ## installation
 
-To install to your Svelte project, open your project's root directory in a terminal. Type `npm i @silintl/ui-components --save-dev` and press enter. You may also have to run `npm i material-components-web@10.0.0  --save-dev` and possibly `npm i sass@1.32.x  --save-dev` to get sass stuff working if it doesn't work initially or you want to develop additional material web components in your app. It may depend on your version of npm if this is necessary. You should already have svelte@3 installed if you are using this library. You will also want to include the following code in your index.html for material icons to work,  `<link href="https://fonts.googleapis.com/icon?family=Material+Icons&display=swap" rel="stylesheet">`. See https://google.github.io/material-design-icons/#sizing for sizing material icons. See the `themeing and typography` section for importing Google Fonts.
+To install to your Svelte project, open your project's root directory in a terminal. Type `npm i @silintl/ui-components --save-dev` and press enter. You may also have to run `npm i material-components-web@10.0.0  --save-dev` and possibly `npm i sass@1.32.x  --save-dev` to get sass stuff working if it doesn't work initially or you want to develop additional material web components in your app. It may depend on your version of npm if this is necessary. You should already have svelte@3 installed if you are using this library.
+
+You will also want to include the following code in your index.html for material icons to work,
+```
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons&display=swap" rel="stylesheet">
```

Comment:
might be a "nit" but the official Material docs use `display=block` and I suspect for icons it's better not to "progressively" enhance those whereas `swap` is a better solution for fonts.

Reply:
Also those 'preconnect' lines I sent you are helpful to have.

Reply:
ah, reading further down now I see the example you provided has the `preconnect` and the `block` as well... this might be confusing though.

Reply:
Good pointers, thanks.